__tests__/app.test.js

```diff
@@ -79,3 +79,38 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+describe("GET /api/articles", () => {
+  test("status 200: should respond with an array of article objects  ", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toBeInstanceOf(Array);
+        expect(body.articles.length).toBe(13);
+        body.articles.forEach((article) => {
+          expect(article).toHaveProperty("author", expect.any(String));
+          expect(article).toHaveProperty("title", expect.any(String));
+          expect(article).toHaveProperty("article_id", expect.any(Number));
+          expect(article).toHaveProperty("topic", expect.any(String));
+          expect(article).toHaveProperty("created_at", expect.any(String));
+          expect(article).toHaveProperty("votes", expect.any(Number));
+          expect(article).toHaveProperty("article_img_url", expect.any(String));
+          expect(article).toHaveProperty("comment_count", expect.any(Number));
+          expect(article.body).toBe(undefined);
+        });
+        expect(body.articles[0].comment_count).toBe(2); //article 3 has 2 comments
+        expect(body.articles[6].comment_count).toBe(11); //article 1 has 11 comments
+        expect(body.articles[0].article_id).toBe(3); //the latest recorded article
+      });
```

Comment:
There is no assertion to check the sort order of the array. You can do this manually but we used jest-sorted in the lecture to help us out with this.

https://www.npmjs.com/package/jest-sorted